src/index.js

```diff
@@ -86,6 +89,14 @@ class CoursesApp extends React.Component {
               {(handleToggle) => (
                 <NavList>
                   <NavListSection>
+                    <NavListItem
+                      id="courses-app-search-item"
+                      onClick={() => {
+                        this.props.history.replace('/cr');
```

Comment:
Why are we using `replace` here rather than the usual `push`? User expectation would normally be that the Back button would take the browser to the previous page, but using the approach coded here it will take the browser back to the page before last.

Maybe there is some subtle requirement here that I've not grokked; but otherwise, I think this should be changed to `push`.